vm/src/frame.rs

```diff
@@ -912,7 +912,9 @@ impl Frame {
             .iter()
             .map(|symbol| vm.ctx.new_str(symbol.to_string()))
             .collect();
-        let module = vm.import(module, &vm.ctx.new_tuple(from_list))?;
+        let level = module.chars().take_while(|char| *char == '.').count();
```

Comment:
Instead of splitting of the level here, it's better to modify the parser to be able to handle this such that have AST nodes with a level inside. This way, the level is already known at parse time, and is passed in as a second argument. See also: https://greentreesnakes.readthedocs.io/en/latest/nodes.html#imports

Reply:
This might be a big change tough, so it can be done in a follow up pull request.

Reply:
Sure. I will do that in a different PR.